http/src/request/channel/message/create_message.rs

```diff
@@ -197,32 +180,6 @@ impl<'a> CreateMessage<'a> {
         Ok(self)
     }
 
-    /// Attach an embed to the message.
-    ///
-    /// Embed total character length must not exceed 6000 characters.
-    /// Additionally, the internal fields also have character limits. Refer to
-    /// [the discord docs] for more information.
-    ///
-    /// # Examples
-    ///
-    /// See [`EmbedBuilder`] for an example.
-    ///
-    /// # Errors
-    ///
-    /// Returns a [`CreateMessageErrorType::EmbedTooLarge`] error type if the
-    /// embed is too large.
-    ///
-    /// [the discord docs]: https://discord.com/developers/docs/resources/channel#embed-limits
-    /// [`EmbedBuilder`]: https://docs.rs/twilight-embed-builder/*/twilight_embed_builder
-    pub fn embed(mut self, embed: Embed) -> Result<Self, CreateMessageError> {
```

Comment:
We'll need another PR to `main` that deprecates all the methods removed by this pr

Reply:
PR'd as #1034.